openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/Component/ProfilerSettingsModal.tsx

```diff
@@ -283,7 +283,7 @@ const ProfilerSettingsModal: React.FC<ProfilerSettingsModalProps> = ({
                   step={1}
                   value={profileSample}
                   onChange={(value) => {
-                    setProfileSample(value);
+                    setProfileSample(value as number);
```

Comment:
```suggestion
                    setProfileSample(Number(value));
```

Let's not force typing and use type casting for safe side

Reply:
Done